openai-core/src/commonMain/kotlin/com.aallam.openai.api/core/RequestOptions.kt

```diff
@@ -8,9 +8,11 @@ import com.aallam.openai.api.http.Timeout
  * @property headers A mutable map of header names to their respective values to be sent with the request.
  * @property urlParameters A mutable map of URL parameter names to their respective values to be appended to the request URL.
  * @property timeout Http operations timeouts.
+ * @property betaVersion chooses beta version of API.
  */
 public data class RequestOptions(
     public val headers: Map<String, String> = emptyMap(),
     public val urlParameters: Map<String, String> = emptyMap(),
     public val timeout: Timeout? = null,
+    public val betaVersion: Int? = null,
```

Comment:
I think It would be better if we update `HttpRequestBuilder.requestOptions` instead, but having something like:
```kotlin
internal fun HttpRequestBuilder.requestOptions(requestOptions: RequestOptions? = null) {
    if (requestOptions == null) return
    requestOptions.headers.forEach { (key, value) ->
        if (headers.contains(key)) {
            headers.remove(key)
        }
        headers[key] = value
    }
    // ...
 }
 ```
 This more future proof, and would overriding any headers later on.

Reply:
After that, simply removing this betaVersion value and changing default beta version for HttpRequestBuilder.beta()  to 2 works well. Thanks!